codecov.yml

```diff
@@ -26,3 +26,4 @@ coverage:
     - "nipype/external/*"
     - "tools/*"
     - "doc/*"
+    - "**/tests"
```

Comment:
we should also add examples to this list. we don't need to see which lines in examples are covered.